text/bandersnatch.tex

```diff
@@ -12,7 +12,7 @@ \section{Bandersnatch Ring VRF}\label{sec:bandersnatch}
 The singly-contextualized Bandersnatch Ring\textsc{vrf} proofs $\bandersnatch{r}{c}{m}$ are a zk-\textsc{snark}-enabled analogue utilizing the Pedersen \textsc{vrf}, also defined by \cite{hosseini2024bandersnatch} and further detailed by \cite{cryptoeprint:2023/002}.
 
 \begin{align}
-  \mathcal{O}(\seq{\H_B}) \in \Y_R &\equiv \text{PCS\_commitment}(\seq{\H_B})  \\
+  \mathcal{O}(\seq{\H_B}) \in \Y_R &\equiv \text{KZG\_commitment}(\seq{\H_B})  \\
```

Comment:
IMO we can be explicit about the used polynomial commitment scheme (even though it doesn't make a lot of difference here and the implementor must refer to the Bandersnatch spec anyway)